docs/report/report.tex

```diff
@@ -45,6 +45,92 @@ \subsubsection{Stretch Goals}
 
 \section{Machbarkeitsstudie}
 
+\subsection{NFC Tag}
+\subsubsection{lesen}
+Um mit NFC Tags arbeiten zu können, müssen diese auch auf entschlüsselt bzw. gelesen werden können.
```

Comment:
was soll 'auch auf entschlüsselt' bedeuten?

Reply:
adressed in commit ebddf4dd3e0ad75eae259bd72ee957a80ea9e62b